src/_incydr_sdk/legal_hold/models.py

```diff
@@ -0,0 +1,183 @@
+from datetime import datetime
+from typing import List
+from typing import Optional
+
+from pydantic import Field
+
+from _incydr_sdk.core.models import ResponseModel
+
+
+class CreatorUser(ResponseModel):
+    user_id: Optional[str] = Field(
```

Comment:
Why are all of the fields marked as Optional? I noticed that is the pattern on existing response models too.